lib/interactor.rb

```diff
@@ -153,14 +153,12 @@ def run!
   # each interactor class.
   #
   # Returns nothing.
-  def call
-  end
+  def call; end
```

Comment:
Rubocop default settings are easily adjustable to project's needs. I'll remove this change.